bin/GameIndex.dbf

```diff
@@ -9223,6 +9223,7 @@ Compat = 3
 	patch=1,EE,001f70b4,word,1000000f
 	// Fixes flickering graphics (Thanks to Kian Stevens for finding this).
 	patch=1,EE,80000000,word,00000000
+	comment=This disc has the same CRC as SLUS-20576, the NTSC-U disc.
```

Comment:
I don't think the comments are necessary, the info is not really important for normal users.
Maybe a code comment under [patches = C5473413] would be better.